src/CarbonAware.WebApi/test/integrationTests/CarbonAwareControllerTests.cs

```diff
@@ -184,39 +186,143 @@ public async Task EmissionsForecastsBatch_MissingRequiredParams_ReturnsBadReques
         Assert.That(result?.StatusCode, Is.EqualTo(HttpStatusCode.BadRequest));
     }
 
-    [Test]
-    public async Task EmissionsForecastsBatch_SupportedDataSources_ReturnsOk()
+    [TestCase("2021-09-01T08:30:00Z", "2021-09-01T09:00:00Z", "2021-09-01T12:00:00Z", "eastus", 1, TestName = "EmissionsForecastsBatch expects OK for eastus 1 element")]
+    [TestCase("2021-09-01T08:30:00Z", "2021-09-01T08:30:00Z", "2021-09-02T08:30:00Z", "westus", 3, TestName = "EmissionsForecastsBatch expects OK for westus 3 element")]
```

Comment:
Can you speak to why we should have two tests here?

Reply:
The goal is to pass more than 1 element. Part of the batchXX endpoints is to handle multiple elements, and there wasn't any of those.